backend/prisma/migrations/20240628172722_init_user_and_trans/migration.sql

```diff
@@ -0,0 +1,47 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "id" SERIAL NOT NULL,
+    "email" TEXT NOT NULL,
+    "password" TEXT NOT NULL,
+    "city" TEXT NOT NULL,
+    "salary" TEXT NOT NULL,
+    "roommates" INTEGER NOT NULL,
+    "children" INTEGER NOT NULL,
+    "job" TEXT NOT NULL,
+
+    CONSTRAINT "User_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Transactions" (
+    "id" SERIAL NOT NULL,
+    "userId" INTEGER NOT NULL,
+    "rent" DOUBLE PRECISION NOT NULL,
+    "utilities" DOUBLE PRECISION NOT NULL,
+    "housing" DOUBLE PRECISION NOT NULL,
+    "loans" DOUBLE PRECISION NOT NULL,
+    "student_loans" DOUBLE PRECISION NOT NULL,
+    "car_loans_and_lease" DOUBLE PRECISION NOT NULL,
+    "credit_card_payments" DOUBLE PRECISION NOT NULL,
+    "other_loans" DOUBLE PRECISION NOT NULL,
+    "entertainment" DOUBLE PRECISION NOT NULL,
+    "streaming_services" DOUBLE PRECISION NOT NULL,
+    "other_entertainment" DOUBLE PRECISION NOT NULL,
+    "food" DOUBLE PRECISION NOT NULL,
+    "resturants" DOUBLE PRECISION NOT NULL,
+    "groceries" DOUBLE PRECISION NOT NULL,
+    "medical_care" DOUBLE PRECISION NOT NULL,
+    "transportation" DOUBLE PRECISION NOT NULL,
+    "gas" DOUBLE PRECISION NOT NULL,
+    "parking" DOUBLE PRECISION NOT NULL,
+    "ride_share" DOUBLE PRECISION NOT NULL,
+    "public_transit" DOUBLE PRECISION NOT NULL,
```

Comment:
What about any transactions that doesn't fit in any of these columns?

Will only one of "rent", "utilities" etc. be filled out per row? If so, it seems like this is a waste of database space (lots of empty columns). Is there a way you could model it so we don't have so many empty columns?